.travis.yml

```diff
@@ -12,6 +12,9 @@ python:
  - "3.4"
  - "pypy"
 install:
+ - if [[ $TRAVIS_PYTHON_VERSION == 2* ]]; then pip install -q python-memcached>=1.57; fi
+ - if [[ $TRAVIS_PYTHON_VERSION == 3* ]]; then pip install -q python3-memcached>=1.51; fi
```

Comment:
Interesting. More interesting that `binascii.crc32(b'foo')` return very different values on python2 and 3. OK, we'll leave this as-is for now.